### PR TITLE
Fix broken image in overflow

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

This PR is w.r.t. #34441 and #20681

### CODE REVIEW

**Review by moT01 — COMMENTED** [review]
This added link is not working for me.

**Comment by thecodingaviator** [comment]
@ahmadabdolsaheb can you please upload the posting image to the CDN and drop a link here?

File: guide/english/css/overflow/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ For example, a given block-level element (`<div>`) set to 300px wide, that conta
   }
 ```
 And the `.box-element` will look like this:
```
Comment (moT01) [inline guide/english/css/overflow/index.md]:
```suggestion
```

**Review by moT01 — CHANGES_REQUESTED** [review]
I'm not sure what I was looking at before, but this does display an image. But it's not on it's own line and it's centered. Removing the line I suggested should put it on its own line I think. Or maybe adding two spaces at the end of the line. If you can make that quick change, we can merge this for now to make the article look better.

**Comment by thecodingaviator** [comment]
@moT01 how does it look now?

**Review by moT01 — APPROVED** [review]
Looks good @thecodingaviator 🎉

**Pull request merged**